components/renderer/Dockerfile

```diff
@@ -3,19 +3,8 @@ FROM node:23.7.0-alpine3.21
 LABEL maintainer="Quality-time team <quality-time@ictu.nl>"
 LABEL description="Quality-time PDF render service"
 
-RUN apk add --no-cache \
-    msttcorefonts-installer=3.8.1-r1 \
-    fontconfig=2.15.0-r1 \
-    font-droid=20200215-r3 \
-    font-noto=24.7.1-r0 \
```

Comment:
Interesting that these font installs were mostly added for the `v3.35` and `v3.36` changes, but their requirement was never explicitly mentioned. Nice that we can get rid of it all :)

Reply:
Many of the examples on how to use Chromium in Docker installed included installation of fonts. I never bothered to check before if it was really needed.